### DISCUSSION TITLE
Bookstack v22.06.2

### DISCUSSION THREAD

**Discussion by addvanced** [hassio-addons/app-bookstack#188] [General]
Hi,

Would it be possible to upgrade bookstack to v22.06.2?
I've made a PR from a fork of your project, upgrading the dependencies - hopefully it can be merged into your main branch.
#187 

If not, hopefully you can upgrade it soon. :)

Thanks for the add-on! I've been hosting it separately for a long time, so it's nice to have it included in HA as an add-on now.

**Comment by frenck** [comment]
Hi!

There is no need to start a discussion, you made a PR already, right?!

../Frenck

**Reply by addvanced** [reply]
Yes, I did.
Sorry, I am not 100% sure about the "process" of doing it.

So... Just started this discussion as a "better safe than sorry".

But, in the future, I'll just made a PR.

/Kenneth
